game/src/constants.js

```diff
@@ -6,7 +6,18 @@ const greetings = [
     'It\'s a plane!',
     '1000% more Cube!',
     'A Whole New Dimension!',
+    'Cubeify everything!'
```

Comment:
It looks like you're missing a `,` here. The game crashes without that.

```suggestion
    'Cubeify everything!',
```